packages/next-swc/crates/core/src/react_remove_properties.rs

```diff
@@ -0,0 +1,72 @@
+use regex::Regex;
+use serde::Deserialize;
+use swc_ecmascript::ast::*;
+use swc_ecmascript::visit::{noop_fold_type, Fold};
+
+#[derive(Clone, Debug, Deserialize)]
+#[serde(untagged)]
+pub enum Config {
+    All(bool),
+    WithOptions(Options),
+}
+
+impl Config {
+    pub fn truthy(&self) -> bool {
+        match self {
+            Config::All(b) => *b,
+            Config::WithOptions(_) => true,
+        }
+    }
+}
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct Options {
+    #[serde(default)]
+    pub properties: Vec<String>,
+}
+
+struct RemoveProperties {
+    properties: Vec<Regex>,
+}
+
+impl RemoveProperties {
+    fn should_remove_property(&self, name: &str) -> bool {
+        self.properties.iter().any(|p| p.is_match(name))
+    }
+}
+
+impl Fold for RemoveProperties {
+    noop_fold_type!();
+
+    fn fold_jsx_opening_element(&mut self, mut el: JSXOpeningElement) -> JSXOpeningElement {
```

Comment:
I think `let el = el.fold_children_with(self)` is required.

Otherwise,

```js

const obj = <Router render={() => (<Page test={true} />})}
```

will not be handled.

will not be handled.

Reply:
Fixed and added test.